docs/actions.md

```diff
@@ -5,28 +5,30 @@ Use actions to update the state.
 ```jsx
 app({
   state: "Hi.",
-  view: (state, actions) => (
-    <h1 onclick={actions.ucase}>{state}</h1>
-  ),
+  view: (state, actions) =>
+    <h1 onclick={actions.ucase}>
+      {state}
+    </h1>,
   actions: {
     ucase: state => state.toUpperCase()
   }
 })
 ```
 
-To be able to update the state, an action must return a new state or a part of it.
+An action must return a new state or a part of it in order to update the state.
```

Comment:
@SkaterDad Added a new section describing how to update a complex state using Ramda! 👍